engine-comparison/run.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+cd
+sudo gcloud docker -- pull gcr.io/fuzzer-test-suite/gcloud-clang-deps
+sudo docker build -t base-image -f ~/input/FTS/engine-comparison/Dockerfile ~/input
```

Comment:
From my understanding of Docker, the final argument of `docker build` is the context; the image can only access files within the context, and it's immutable after image creation. 

A good section of a page I spent some (haha, lots of) time reading is https://docs.docker.com/engine/reference/builder/#copy

> The <src> path must be inside the context of the build; you cannot COPY ../something /something, because the first step of a docker build is to send the context directory (and subdirectories) to the docker daemon.